src/components/ui/button/index.tsx

```diff
@@ -1,8 +1,73 @@
 import { forwardRef } from 'react';
 import { Slot } from '@radix-ui/react-slot';
-import { type VariantProps } from 'class-variance-authority';
+import { type VariantProps, cva } from 'class-variance-authority';
 import { cn } from 'Utils/cn';
-import { buttonVariants } from './variants';
+
+export const buttonVariants = cva(
```

Comment:
Do u need to export this?
```suggestion
const buttonVariants = cva(
```

Reply:
Yep, we can apply button's style to other component/element with this buttonVariants

<img width="917" alt="Screenshot 2023-11-01 at 10 25 58 AM" src="https://github.com/deriv-com/smarttrader/assets/126637868/895408c7-bb91-4fa8-a7dd-f933f90d2ead">